File: Lua/Exercício 1.lua
print("Digite o valor inicial: ")
local inicio = tonumber(io.read())

print("Digite o valor final: ")
local fim = tonumber(io.read())

print("Digite o número para verificar os múltiplos: ")
local divisor = tonumber(io.read())

for numero = inicio, fim, 1 do --start, end, step
  if numero % divisor == 0 then
    print(numero)
  end
end

File: Go/Exercício 1.go
package main
import "fmt"

func avaliarValor(valor int) string {
	if valor > 0 {
		return "Positivo"
	} else if valor < 0 {
		return "Negativo"
	} else {
		return "Zero"
	}
}

func main() {
	var entrada int

	fmt.Print("Digite um número: ")
	fmt.Scanln(&entrada)

	fmt.Println(avaliarValor(entrada))
}
